src/loaders/lottie/tvgLottieModifier.h

```diff
@@ -40,4 +41,34 @@ struct RoundnessModifier
 };
 
 
+struct OffsetPathModifier
+{
+    float offset = 0.0f;
+    float miterLimit = 4.0f;
+    StrokeJoin join = StrokeJoin::Miter;
+
+    OffsetPathModifier(float offset, float miter, StrokeJoin join) : offset(offset), miterLimit(miter), join(join) {};
+
+    OffsetPathModifier(const OffsetPathModifier* off) {
+        *this = *off;
+    }
+
+    void appendPath(const PathCommand* inCmds, uint32_t inCmdsCnt, const Point* inPts, uint32_t inPtsCnt, Array<PathCommand>& outCmds, Array<Point>& outPts, bool clockwise) const;
```

Comment:
please consider the method names as well as mentioned in roundness.

Reply:
please check now, 2 of them are wrappers for modifyPath